lib/colorls/core.rb

```diff
@@ -184,7 +182,6 @@ def mode_info(stat)
           end
         end
       end
-      mode
```

Comment:
@athityakumar 

This change broke the Mode view

it now only displays 1's and 0's.

Please revert.

Reply:
@GladOSkar - Can you add a small PR fixing the buggy parts of this PR?